### ISSUE TITLE
Auto populate local-only folder game metadata

### ISSUE THREAD

**Issue by mrmwiebe** [winnitron/WinnitronLauncher#36]
Want to make it so that if no metadata is found for a local folder game (maybe even in some cases for downloaded? thoughts @aaronklaassen ) the launcher attempts to auto-generate a metadata file if it can find everything.

**Comment by aaronklaassen** [comment]
Yeah for local games for sure that makes sense. Makes it more clear to the owner how to build that file if they're just editing an existing one.

**Comment by aaronklaassen** [comment]
Actually going to save this for 2.4.0 - do it as part of the big sync and game folder reorganization (#32).

**Comment by mrmwiebe** [comment]
Hey @aaronklaassen It's been a while since we posted this, has this been addressed?  Can we close it?

**Comment by aaronklaassen** [comment]
@mrmwiebe No, I never did this. Still a good idea though I think.
